src/parameter_declarations.ts

```diff
@@ -325,3 +327,14 @@ Parameter.declare({
 		'(OPTIONAL) Derives a drive key for the given drive ID without verifying its correctness against the drive on chain.',
 	type: 'boolean'
 });
+
+Parameter.declare({
+	name: LocalPathsParameter,
+	aliases: ['--local-paths'],
+	type: 'array',
+	description: `(BETA) a glob pattern to select files to be uploaded
+\t\t\t\t\t\t\t• Can NOT be used in conjunction with --local-files-path
+\t\t\t\t\t\t\t• Can NOT be used in conjunction with --local-files
+\t\t\t\t\t\t\t• Can lead to false positives or upload errors when glob string is not surrounded by quotation marks`,
+	forbiddenConjunctionParameters: [LocalFilesParameter, LocalFilePathParameter]
```

Comment:
`DestinationFileNameParameter` should probably be disallowed as well, though one could argue that it's valid in the single file case.